tests/posit/connect/test_resources.py

```diff
@@ -1,8 +1,53 @@
 import pytest
+import warnings
 
 from typing import Any, List, Optional
 
-from posit.connect.resources import Resources
+from posit.connect.resources import Resource, Resources
+
+
+class FakeResource(Resource):
+    @property
+    def foo(self) -> str | None:
+        return self.get("foo")
+
+
+class TestResource:
+    def test__getitem__(self):
+        warnings.filterwarnings("ignore", category=FutureWarning)
```

Comment:
We should move this to a global test suite setup to reduce noise.

Reply:
Why? Our tests shouldn't generally be raising this warning, except when we're testing explicitly for it, right?

Reply:
Any test which invoke `Resource#__get_item__` will display the warning message. However, the warnings module is designed to only show the warning message once per argument set.

Reply:
Right, I just figured that if we were calling getitem in a test for some reason, we should also assert the warning in the test